drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -72,7 +72,18 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMediumUnspecified:
 #endif
                     case NdisPhysicalMediumWirelessLan:
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Comment:
> select at runtime
 
 If such behavior is wanted, maybe investigate/do this NT6+ enhancement as a follow-up?
 
 > _WIN32_NINNT and NTDDI_VERSION
 
 No: these are compile-time only.

Reply:
>If such behavior is wanted, maybe investigate/do this NT6+ enhancement as a follow-up?

Yep, it's needed for NT6 target still, to have networking working properly in that case too, but I doubt I can implement it soon, because it might contain a lot of changes and needs much more time to work on this.

>No: these are compile-time only.

Oh, OK. Then runtime option should be handled differently. And I'm not sure yet how.